src/common/config/index.tsx

```diff
@@ -5,38 +5,48 @@
   3. production (yarn build)
 */
 
-const ENV = "development";
+import getEnv from "./getEnv";
 
-const development = {
-  BASE_API: "http://localhost:5000",
+const ENV = getEnv();
+
+const local = {
+  API_URL: "http://localhost:5000",
+  CLIENT_URL: "http://localhost:3000",
+  AUTH0_DOMAIN: "dev-qlg7c9vz.us.auth0.com",
+  AUTH0_CLIENT_ID: "czQVb2eCU05djAnNy96MAh5Ki9JxKY2e",
 };
 
-const production = {
-  ...development,
+const development = {
+  ...local,
+  CLIENT_URL: "https://chatbotish-dash-dev.vercel.app",
 };
 
 const staging = {
   ...development,
+  CLIENT_URL: "https://chatbotish-dash.vercel.app",
```

Comment:
Is this the correct staging `CLIENT_URL`?

Reply:
done